src/commands/prune.cr

```diff
@@ -1,19 +1,20 @@
 require "file_utils"
 require "./command"
+require "../helpers/files"
```

Comment:
This file should be moved:

```suggestion
require "../helpers"
```

Reply:
It doesn't really matter either way.

Reply:
IMO it does from the good practices standpoint - i.e. naming files according to their namespace path `Foo::Bar::Baz` -> `foo/bar/baz.cr`

Reply:
Yes, that does actually seem better. Will include this change. No idea

Reply:
@Sija In general, that's true. But for this use case not so much IMO. A helpers namespace is intended to contain a mixture of different things (eventually). Otherwise you would use a more specifc namespace. When there are lots of helpers you may not want to place them in a single file at `./helpers` but separted by concerns in `./helpers/*`. Currently, we don't need that. But maybe in the future. It can be adapter later when necessary, but it wouldn't hurt to leave it as it was.
The change from `helpers/files` to `helper` IMO didn't help to make the PR any better, yet it requires another round of reviews (quick ones, obviously) when it could have just been merged as it was. 🤷‍♂️